_posts/2025-11-14-tholin-feature.md

```
@@ -0,0 +1,149 @@
+
+---
+
+### Multi-Project Die Exploration
+
+{% include post-gallery.html gallery="multi_project" %}
+
+
+Tholin is also pioneering **multi-project die (MPD)** strategies, maximizing wafer area by integrating multiple small designs onto a single die.
+Projects such as [`gfmpw-1-multi`](https://github.com/AvalonSemiconductors/gfmpw1-multi) and [`AS2650-bring-up`](https://github.com/AvalonSemiconductors/AS2650-bring-up) demonstrate how careful planning and padframe reuse make it possible to test multiple subsystems in one run.
+
+Tholin has also written thorough documentation on this project [here](https://avalonsemiconductors.github.io/GFMPW-1-MULTI/index.html)
```

markdownlint-cli2 (0.21.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)